packages/core/src/components/graphs/gauge.ts

```diff
@@ -121,14 +121,21 @@ export class Gauge extends Component {
 
 		// Add data arc
 		const arcValue = svg.selectAll("path.arc-foreground").data([value]);
+		const userProvidedScale = Tools.getProperty(options, "color", "scale");
+		const noProvidedColorScale = userProvidedScale === null || Object.keys(userProvidedScale).length === 0;
 
 		arcValue
 			.enter()
 			.append("path")
-			.attr("class", "arc-foreground")
+			.attr(
+				"class",
+				(d) => noProvidedColorScale
+					? `arc-foreground ${self.model.getColorClass(d[groupMapsTo])}`
+					: "arc-foreground"
+			)
+			.attr("fill", (d) => noProvidedColorScale ? null : self.model.getFillColor(d[groupMapsTo]))
```

Comment:
we need to find a solution to this, it's not ideal to have every graph check for the custom color scale